packages/sei-cosmwasm/src/sei_types.rs

```diff
@@ -54,3 +54,68 @@ pub struct OrderResponse {
     pub position_direction: PositionDirection,
     pub data: String,
 }
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+pub struct SettlementEntry {
+    pub account: String,
+    pub price_denom: String,
+    pub asset_denom: String,
+    pub quantity: Decimal,
+    pub execution_cost_or_proceed: Decimal,
+    pub expected_cost_or_proceed: Decimal,
+    pub position_direction: PositionDirection,
+    pub order_type: OrderType,
+    pub order_id: u64,
+}
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+pub struct DepositInfo {
+    pub account: String,
+    pub denom: String,
+    pub amount: Decimal,
+}
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+pub struct LiquidationRequest {
+    pub requestor: String,
+    pub account: String,
+}
+
+#[derive(Serialize, Deserialize, Clone, PartialEq, JsonSchema)]
+pub struct LiquidationResponse {
+    pub successful_accounts: Vec<String>,
+    pub liquidation_orders: Vec<Order>,
+}
+
+#[derive(Serialize, Deserialize, Clone, PartialEq, JsonSchema)]
+pub struct BulkOrderPlacementsResponse {
+    pub unsuccessful_orders: Vec<UnsuccessfulOrder>,
+}
+
+#[derive(Serialize, Deserialize, Clone, PartialEq, JsonSchema)]
+pub struct UnsuccessfulOrder {
+    pub id: u64,
+    pub reason: String,
+}
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+pub struct ContractOrderResult {
```

Comment:
heads up, this is already included in sei_types from my previous PR, so you'll probably have to handle that rebase conflict